utilities/transactions/write_prepared_txn_db.cc

```diff
@@ -798,7 +798,7 @@ void WritePreparedTxnDB::UpdateSnapshots(
   // afterwards.
   size_t i = 0;
   auto it = snapshots.begin();
-  for (; it != snapshots.end() && i < SNAPSHOT_CACHE_SIZE; it++, i++) {
+  for (; it != snapshots.end() && i < SNAPSHOT_CACHE_SIZE; ++it, ++i) {
```

Comment:
It's not wrong but `i++` => `++i` is not mandated.

Reply:
Afaik pre-increment is preferred in all cases but not mandatory for scalar types (as you said). Did this for consistency within same code line, but I'm fine either way.

Reply:
I'm fine with either way too.
In my understganding, the style guideline mentioned the "con" part about pre-increment so that it doesn't offer any preference for "simple scalar (non-object) values".